src/scripts/main.js

```diff
@@ -21,7 +21,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const item in data) {
+    const li = document.createElement('li');
+
+    li.textContent = item;
+
+    if (Object.values(data[item]).length > 0) {
+      element.append(li);
+
+      const ul = document.createElement('ul');
+
+      li.append(ul);
+      createTree(ul, data[item]);
+    }
+    element.append(li);
```

Comment:
The `li` element is appended to `element` twice: once on line 30 and again on line 37. This redundancy can cause the same `li` to appear twice in the DOM. You should remove the `element.append(li);` on line 37 to fix this issue.